### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
added alert component
image

### CODE REVIEW

File: css/component_css/alert.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    border-radius: 4px;
+
+}
+
+.icon{
+    margin-right: 5px;
+}
+
+.alert-success{
+    background-color: rgb(189 255 186);
```
Comment (znjs) [inline css/component_css/alert.css:28]: if possible try using variable names instead of RGB or hex

File: css/component_css/drawer.css [hunk 2]
```diff
@@ -145,4 +129,10 @@ body{
 .btn:hover{
     cursor: pointer;
     background-color: rgba(0,0,0,0.1);
+}
+
+.documentation{
+    margin-left: 260px;
```
Comment (znjs) [inline css/component_css/drawer.css]: try using relative units, it helps in making it responsive

**Review by znjs — APPROVED** [review]
👍

**Review by Pratik1005 — APPROVED** [review]

File: css/component_css/drawer.css (hunk 2 above)
Comment (mohammadkhizerkhan) [reply css/component_css/drawer.css]: thanks jaya, i'll fix it

**Pull request merged**